src/Core/Services/Implementations/OrganizationService.cs

```diff
@@ -659,10 +715,21 @@ public async Task VerifyBankAsync(Guid organizationId, int amount1, int amount2)
             PrivateKey = signup.PrivateKey,
             CreationDate = DateTime.UtcNow,
             RevisionDate = DateTime.UtcNow,
-            Status = OrganizationStatusType.Created
+            Status = OrganizationStatusType.Created,
+            UsePasswordManager = true
         };
 
-        if (plan.Type == PlanType.Free && !provider)
+        var secretsManagerPlan = plans.FirstOrDefault(x => x.BitwardenProduct == BitwardenProductType.SecretsManager);
+        if (secretsManagerPlan != null && signup.UseSecretsManager)
+        {
+            organization.SmSeats = (short)(secretsManagerPlan.BaseSeats + signup.AdditionalSmSeats);
+            organization.SmServiceAccounts =
+                (short)(secretsManagerPlan.BaseServiceAccount + signup.AdditionalServiceAccount);
+            organization.UseSecretsManager = true;
+        }
```

Comment:
See feedback above, you should do the same thing here (whatever we end up doing)